### DISCUSSION TITLE
Muti level swagger docs

### DISCUSSION THREAD

**Discussion by chuxiangfeng** [fastapi/fastapi#8376] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
when I use tags to classify endpoints, swagger ui docs only has one level display, if i want Muti-level api docs, how can I do ?
```

### Description

when I use tags to classify endpoints, swagger ui docs only has one level display, if i want Muti-level api docs, how can I do ?

### Operating System

macOS

### FastAPI Version

0.79.0

### Python Version

3.7.13

### Additional Context

![](https://fastapi.tiangolo.com/img/index/index-03-swagger-02.png)

like this， https://stackoverflow.com/questions/41800330/multi-level-nested-tagging-in-swagger-ui

**Comment by facundopadilla** [comment]
Hello, do you need something like that?: https://pypi.org/project/fastapi-versioning/

**Comment by chuxiangfeng** [comment]
> Hello, do you need something like that?: https://pypi.org/project/fastapi-versioning/

I want like this， https://stackoverflow.com/questions/41800330/multi-level-nested-tagging-in-swagger-ui

**Comment by JarroVGIT** [comment, marked as answer]
You point to a Stackoverflow question with an answer that explains that it is not possible. Doesn’t that answer your question?

**Reply by YuriiMotov** [reply]
BTW, it will be supported in openapi 3.2 https://github.com/OAI/OpenAPI-Specification/issues/1367#issuecomment-2619142072
